xrayutilities/materials/predefined_materials.py

```diff
@@ -32,186 +32,186 @@
+Ag = Crystal("Ag", lattice=SGLattice(225, 4.0853, atoms=[e.Ag, ], pos=['4a', ]))
+SnAlpha = Crystal("Sn-alpha", lattice=SGLattice('227:1', 6.4912, atoms=[e.Sn, ],
                                         pos=['8a', ]))
-Cu = Crystal("Cu", SGLattice(225, 3.61496, atoms=[e.Cu, ], pos=['4a', ]))
-CaTiO3 = Crystal("CaTiO3", SGLattice(221, 3.795, atoms=[e.Ca, e.Ti, e.O],
+Cu = Crystal("Cu", lattice=SGLattice(225, 3.61496, atoms=[e.Cu, ], pos=['4a', ]))
+CaTiO3 = Crystal("CaTiO3", lattice=SGLattice(221, 3.795, atoms=[e.Ca, e.Ti, e.O],
                                      pos=['1a', '1b', '3c']))
-SrTiO3 = Crystal("SrTiO3", SGLattice(221, 3.905, atoms=[e.Sr, e.Ti, e.O],
+SrTiO3 = Crystal("SrTiO3", lattice=SGLattice(221, 3.905, atoms=[e.Sr, e.Ti, e.O],
                                      pos=['1a', '1b', '3c']))
-# BiFeO3 = Crystal("BiFeO3", SGLattice())
+# BiFeO3 = Crystal("BiFeO3", lattice=SGLattice())
 # BiFeO3 = Crystal(
 #    "BiFeO3",
 #    lattice.PerovskiteTypeRhombohedral(elements.Bi, elements.Fe, elements.O,
 #                                       3.965, 89.3))
-FeO = Crystal("FeO", SGLattice(225, 4.332, atoms=[e.Fe, e.O],
+FeO = Crystal("FeO", lattice=SGLattice(225, 4.332, atoms=[e.Fe, e.O],
                                pos=['4a', '4b']))
-CoO = Crystal("CoO", SGLattice(225, 4.214, atoms=[e.Co, e.O],
+CoO = Crystal("CoO", lattice=SGLattice(225, 4.214, atoms=[e.Co, e.O],
                                pos=['4a', '4b']))
-Fe3O4 = Crystal("Fe3O4", SGLattice('227:2', 8.3958, atoms=[e.Fe, e.Fe, e.O],
+Fe3O4 = Crystal("Fe3O4", lattice=SGLattice('227:2', 8.3958, atoms=[e.Fe, e.Fe, e.O],
                                    pos=['8a', '16d', ('32e', 0.255)]))
-Co3O4 = Crystal("Co3O4", SGLattice('227:2', 8.0821, atoms=[e.Co, e.Co, e.O],
+Co3O4 = Crystal("Co3O4", lattice=SGLattice('227:2', 8.0821, atoms=[e.Co, e.Co, e.O],
                                    pos=['8a', '16d', ('32e', 0.255)]))
-FeRh = Crystal("FeRh", SGLattice(221, 2.993, atoms=[e.Fe, e.Rh],
+FeRh = Crystal("FeRh", lattice=SGLattice(221, 2.993, atoms=[e.Fe, e.Rh],
                                  pos=['1a', '1b']))
-Ir20Mn80 = Crystal("Ir20Mn80", SGLattice(225, 3.780, atoms=[e.Ir, e.Mn],
+Ir20Mn80 = Crystal("Ir20Mn80", lattice=SGLattice(225, 3.780, atoms=[e.Ir, e.Mn],
                                          pos=['4a', '4a'], occ=[0.2, 0.8]))
-CoFe = Crystal("CoFe", SGLattice(221, 2.8508, atoms=[e.Co, e.Fe],
+CoFe = Crystal("CoFe", lattice=SGLattice(221, 2.8508, atoms=[e.Co, e.Fe],
                                  pos=['1a', '1b']))
-LaB6 = Crystal("LaB6", SGLattice(221, 4.15692, atoms=[e.La, e.B],
+LaB6 = Crystal("LaB6", lattice=SGLattice(221, 4.15692, atoms=[e.La, e.B],
                                  pos=['1a', ('6f', 0.19750)]))
-Al2O3 = Crystal("Al2O3", SGLattice('167:H', 4.7602, 12.9933,
+Al2O3 = Crystal("Al2O3", lattice=SGLattice('167:H', 4.7602, 12.9933,
                 atoms=[e.Al, e.O], pos=[('12c', 0.35216), ('18e', 0.30624)]))
```

Comment:
please explain the motivation for this code rearrangement?

Reply:
Can't recall...